### PR TITLE
Tab items/bottom border

### PR DESCRIPTION
**Pull request by nabil-k**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Added a bottom border

### Verification

(how the change was tested, including both manual and automated tests)

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: packages/utils/tokens/src/border-tokens.ts [hunk 1]
```diff
@@ -9,13 +9,15 @@ export interface IBorderTokens {
...
 export const borderTokens: IOperationSet<IBorderTokens, ITheme> = [
   { source: 'borderColor', lookup: getPaletteFromTheme },
   { source: 'borderWidth' },
   { source: 'borderRadius' },
   { source: 'borderStyle' },
+  { source: 'borderBottomWidth' },
 ];
 
-export const borderStyles = tokenBuilder<IBorderTokens>('borderColor', 'borderRadius', 'borderStyle', 'borderWidth');
+export const borderStyles = tokenBuilder<IBorderTokens>('borderColor', 'borderRadius', 'borderStyle', 'borderWidth'); //'borderBottomWidth');
```
Comment (dake3601) [inline packages/utils/tokens/src/border-tokens.ts]: Why is this commented out?

File: packages/components/Tabs/src/TabsItem.types.ts [hunk 2]
```diff
@@ -5,8 +5,9 @@ import { ITextProps } from '@fluentui-react-native/text';
 import { IFocusable } from '@fluentui-react-native/interactive-hooks';
 import type { IViewWin32Props } from '@office-iss/react-native-win32';
 import { FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
-// import { IPressableProps } from '@fluentui-react-native/pressable';
+//import { IPressableProps } from '@fluentui-react-native/pressable';
 import { IButtonProps } from '../../Button/lib';
+import { ViewStyle } from 'react-native';
```
Comment (dake3601) [inline packages/components/Tabs/src/TabsItem.types.ts]: Where do you use this?

File: packages/components/Tabs/src/TabsItem.types.ts [hunk 3]
```diff
@@ -45,7 +46,6 @@ export interface TabsItemTokens extends FontTokens, IForegroundColorTokens, IBac
 export interface TabsItemSlotProps {
   root: React.PropsWithRef<IViewWin32Props>;
   button: ViewProps;
-  content: ITextProps;
```
Comment (dake3601) [inline packages/components/Tabs/src/TabsItem.types.ts]: Does this affect the bottom border?

**Review by dake3601 — CHANGES_REQUESTED** [review]
TabItems.types.ts does not seem to affect the bottom border, could you revert the changes on that file?

File: packages/utils/tokens/src/border-tokens.ts (hunk 1 above)
Comment (nabil-k) [reply packages/utils/tokens/src/border-tokens.ts]: No reason at all, just an accident from when we were debugging w/ krystal earlier

**Pull request merged**